src/providers/bttv/BttvEmotes.cpp

```diff
@@ -138,16 +139,41 @@ void BttvEmotes::loadEmotes()
         .execute();
 }
 
-void BttvEmotes::loadChannel(const QString &channelId,
+void BttvEmotes::loadChannel(TwitchChannel &channel, const QString &channelId,
                              std::function<void(EmoteMap &&)> callback)
 {
     NetworkRequest(QString(bttvChannelEmoteApiUrl) + channelId)
         .timeout(3000)
-        .onSuccess([callback = std::move(callback)](auto result) -> Outcome {
-            auto pair = parseChannelEmotes(result.parseJson());
-            if (pair.first)
-                callback(std::move(pair.second));
-            return pair.first;
+        .onSuccess(
+            [callback = std::move(callback), &channel](auto result) -> Outcome {
+                auto pair = parseChannelEmotes(result.parseJson());
+                if (pair.first)
+                    callback(std::move(pair.second));
+                channel.addMessage(makeSystemMessage("BTTV: emotes reloaded."));
+                return pair.first;
+            })
+        .onError([channelId, &channel](auto result) {
+            if (result.status() == 203)
+            {
+                // User does not have any BTTV emotes
+                channel.addMessage(
+                    makeSystemMessage("BTTV: this channel has no emotes."));
```

Comment:
For any public-facing messages, I think we should stick to the preferred branding of the service. Here it's "BetterTTV". and making the messages just a proper sentence (e.g. "This channel has no BetterTTV channel emotes") feels better to me.

Reply:
Fixed.